test/unit/kubernetes-deploy/options_helper_test.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+require 'test_helper'
+require 'tempfile'
+
+class OptionsHelperTest < KubernetesDeploy::TestCase
+  def test_single_template_dir_only
```

Comment:
let's add one for when the template dir is defaulted based on `$ENVIRONMENT`--that case should also return a non-temp path